_sass/layout/details-summary.scss

```diff
@@ -0,0 +1,3 @@
+details > summary > p {
+  display: inline;
```

Comment:
This is to make sure the little arrow is on the same line as "See inlining steps".